R/parasite_parameters.R

```diff
@@ -0,0 +1,233 @@
+#' @title Parameterise the parasite species to use in the model
+#'
```

Comment:
Could we have a citation somewhere in this description. It's useful when others want to write up their S.Is

Reply:
Good point! I've added a citation.